ebu_tt_live/documents/test/test_converters.py

```diff
@@ -201,10 +201,12 @@ def test_ericsson_smpte_with_overridden_start_of_programme(self):
             use_doc_id_as_seq_id=True,
             smpte_start_of_programme='11:00:00:00')
 
-    def test_ericsson_smpte_with_overridden_start_of_programme_and_sub_zero(self):
+    def test_ericsson_smpte_with_overridden_start_of_programme_and_sub_zero(
+            self):
 
         xml_file = self._load_asset(
-            'converter_ericsson1_smpte_with_start_of_programme_and_sub_zero.xml')
+            'converter_ericsson1_smpte_with_start_of_programme_and_sub_zero'
+            '.xml')
```

Comment:
This is why I'm not a fan of the 80 character line length limit. Splitting a string like this is kind of confusing.
In particular, if I'm searching for usages of that particular file, I would go into my editor and search for `converter_ericsson1_smpte_with_start_of_programme_and_sub_zero.xml`. This line won't come up in the search results anymore

Reply:
I know, it's really annoying. But https://www.python.org/dev/peps/pep-0008/#maximum-line-length says 79 characters! But also:

> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the line length limit up to 99 characters, provided that comments and docstrings are still wrapped at 72 characters.

So I don't mind increasing it...

Reply:
I'm going to resolve this by making it a single long line and adding `  # noqa:E501` to the line to make flake8 let me off the hook here.

Reply:
Done in 3f3cca8